### PR TITLE
移除在 ruby 2.4.x 中已经被废弃的 Fixnum 的用法

### PR DESCRIPTION
**Pull request by bachue**
支持 ruby 2.4.x

### CODE REVIEW

**Comment by songjiayang** [comment]
@bachue  你在负责 Ruby SDK 维护吗？ 

rubygems 发布的版本是 6.8.1，  是否可以把修改放在 6.9.0 中，最后做一个大的 release ？

File: lib/qiniu/auth.rb [hunk 1]
```diff
@@ -171,7 +171,7 @@ def authorize_download_url(url, args = EMPTY_ARGS)
 
           ### URL变换：追加FOP指令
           if args[:fop].is_a?(String) && args[:fop] != '' then
-            if download_url.index('?').is_a?(Fixnum) then
+            if download_url.index('?')
```
Comment (songjiayang) [inline lib/qiniu/auth.rb]: how about  using `include? `

File: lib/qiniu/version.rb [hunk 2]
```diff
@@ -4,7 +4,7 @@ module Qiniu
     module Version
       MAJOR = 6
       MINOR = 9
-      PATCH = 0
+      PATCH = 1
```
Comment (songjiayang) [inline lib/qiniu/version.rb]: v6.9.0 hadn't released yet and we can merge more changes, do the release with another pr ?

**Comment by bachue** [comment]
@songjiayang 不是 我只是替意见

File: lib/qiniu/version.rb (hunk 2 above)
Comment (bachue) [inline lib/qiniu/version.rb]: 可以随 v6.9.0 发布，没有问题

File: lib/qiniu/auth.rb (hunk 1 above)
Comment (bachue) [inline lib/qiniu/auth.rb]: 可以

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -3,6 +3,7 @@
 require 'bundler/setup'
 require 'qiniu'
 require 'rspec'
+require 'webmock'
```
Comment (songjiayang) [inline spec/spec_helper.rb]: Maybe you just need `require 'webmock/rsepc'`

Comment (bachue) [inline spec/spec_helper.rb]: @songjiayang  I tried this at the very beginning, and then I found not all specs requires this but I had to disable it in each spec. And DSLs defined in `webmock/rspec` are not as powerful as native APIs, that's why I still use native APIs.

Comment (songjiayang) [inline spec/spec_helper.rb]: got it

**Comment by songjiayang** [comment]
It's so cool
